src/backend/EdFi.DataManagementService.Backend.Postgresql/Operation/LockOption.cs

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+namespace EdFi.DataManagementService.Backend.Postgresql.Operation;
+
+/// <summary>
+/// An option that tells a SQL action whether to acquire a pessimistic lock
+/// </summary>
+public enum LockOption
```

Comment:
I ran the bulk loader to see what was up with getting serialization failures on non-conflicting inserts. Short version is that it happens, PostgreSQL is being conservative in that isolation mode, and repeatable read isolation plus a little locking on our part is more surgical.